src/screens/Authentication/AdminRegister/AdminRegister.tsx

```diff
@@ -62,21 +107,34 @@ function AdminRegisterScreen({
                 name="email"
                 label="email"
                 placeholder=" email@email.com"
-                onChangeText={setEmail}
+                hasError={emailError !== ''}
+                onChangeText={emailInput => handleEmailChange(emailInput)}
               />
+              {emailError !== '' && (
+                <Text style={{ color: 'red' }}>{emailError}</Text>
+              )}
               <Text style={styles.smallText}>Create Password </Text>
               <AuthInput
                 name="password"
                 label="password"
                 placeholder=" password"
-                onChangeText={setPassword}
+                hasError={passwordError !== ''}
+                onChangeText={passInput => handlePasswordChange(passInput)}
               />
               <Text style={styles.smallText}>Verify Password </Text>
               <AuthInput
                 name="confirmPassword"
                 label="confirmPassword"
                 placeholder=" password"
+                hasError={(passwordError !== '') || (confirmError !== '')}
+                onChangeText={setConfirmPass}
               />
+              {passwordError !== '' && (
+                <Text style={{ color: 'red' }}>{passwordError}</Text>
+              )}
+              {confirmError !== '' && (
```

Comment:
I think you can encapsulate both password strength errors and the error if the passwords don't match in passwordError